docs/Lua API.rst

```diff
@@ -4215,7 +4215,7 @@ Native functions (exported to Lua)
 
   adds a number to the sequence
 
-- ``ShuffleSequence(rngID, seqID)``
+- ``ShuffleSequence(seqID, rngID)``
```

Comment:
since this is an established API, we should avoid changing it, even if the new parameter order would look more consistent. There is no need to break existing clients. Changing APIs is the kind of thing that frustrates contributors, and should not be done lightly.

On the other hand, if you could make either order of parameters work (so as not to break existing clients), we could change the "preferred" ordering here without problems.

Reply:
> On the other hand, if you could make either order of parameters work (so as not to break existing clients), we could change the "preferred" ordering here without problems.

done

Reply:
For clarity, I meant it will try to detect which type of id each argument is and swap them if it thinks they're in reverse. The downside: things get weird [buggy] before filling in all 16bits of the id space(s)